src/libs/actions/IOU.ts

```diff
@@ -7389,6 +7403,40 @@ function cancelPayment(expenseReport: OnyxEntry<OnyxTypes.Report>, chatReport: O
     );
 }
 
+function completePaymentOnboarding(paymentSelected: ValueOf<typeof CONST.PAYMENT_SELECTED>) {
+    const isInviteOnboardingComplete = introSelected?.isInviteOnboardingComplete ?? false;
+
+    if (isInviteOnboardingComplete || !introSelected?.choice) {
+        return;
+    }
+
+    const session = SessionUtils.getSession();
+
+    const personalDetailsListValues = Object.values(
+        OptionsListUtils.getPersonalDetailsForAccountIDs(session?.accountID ? [session.accountID] : [], personalDetailsList),
+    ) as OnyxTypes.PersonalDetails[];
```

Comment:
Let's change the return type in `getPersonalDetailsForAccountIDs` to `SetNonNullable<PersonalDetailsList>`. This way you can remove the assertion here:

```suggestion
    const personalDetailsListValues = Object.values(
        OptionsListUtils.getPersonalDetailsForAccountIDs(session?.accountID ? [session.accountID] : [], personalDetailsList),
    );
```